src/openzaak/components/autorisaties/admin.py

```diff
@@ -50,13 +51,20 @@ def _get_extra(self, obj) -> str:
                 "<strong>Maximale vertrouwelijkheidaanduiding</strong>: "
                 "{va}"
             )
-            zaaktype = get_related_object(obj)
+            try:
```

Comment:
This try-except is actually wrong if you happen to have UUIDs that are the same, as I just experienced during testing.

I have a catalogue that was imported with the same UUIDs (somehow), but the full URLs are obviously different. So while I'm configuring access on a zaaktype from https://catalogi.vng.cloud/api/v1/, the admin overview is linking me to the internal zaaktype.

This resolution should not be tried, instead, it should use the "check if local object based on URL" mechanism. It's implemented in `django_loose_fk.loaders.BaseLoader.is_local_url`